pepr.ts

```diff
@@ -0,0 +1,35 @@
+import { Capability, Log, PeprModule } from "pepr";
```

Comment:
❤️ will try to digest the pepr changes once this is all out of draft/after the weekend.

Reply:
Yeah big idea is leverage monorepo for pepr -- but have a way to tell pepr to bootstrap only one "uds capability" at a time for testing.